src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,14 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(",")))
+                .distinct()
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() ->
+                        new RuntimeException("Can't get min value from list: method_input_list"));
```

Comment:
```suggestion
                        new RuntimeException("Can't get min value from list: " + numbers));
```
let's add our input to message to make it more informative